prow/config/config_test.go

```diff
@@ -333,11 +333,11 @@ periodics:
 			expectError: true,
 		},
 		{
-			name: "with bad repo config",
+			name: "repo should inherit from default config",
```

Comment:
Why are we changing a test rather than adding a new one?

Reply:
because this test case is wrong.

Reply:
also, this test case is the reason why #17346 passed.

Reply:
> also, this test case is the reason why #17346 passed.

Soryr, I don't understand? I think the reason #17346 passed is that there was no test for the inheriting

Reply:
Indeed. If there were tests for the inheritance it would have fail.